src/sys/ptrace.rs

```diff
@@ -6,68 +6,87 @@ use libc::{self, c_void, c_long, siginfo_t};
 use ::unistd::Pid;
 use sys::signal::Signal;
 
-pub mod ptrace {
-    use libc::c_int;
-
-    cfg_if! {
-        if #[cfg(any(all(target_os = "linux", arch = "s390x"),
-                     all(target_os = "linux", target_env = "gnu")))] {
-            pub type PtraceRequest = ::libc::c_uint;
-        } else {
-            pub type PtraceRequest = c_int;
-        }
+
+cfg_if! {
+    if #[cfg(any(all(target_os = "linux", arch = "s390x"),
+                all(target_os = "linux", target_env = "gnu")))] {
+        pub type RequestType = ::libc::c_uint;
+    } else {
+        pub type RequestType = ::libc::c_int;
+    }
+}
+
+libc_enum!{
+    #[cfg_attr(all(any(all(target_os = "linux", arch = "s390x"),
```

Comment:
Ca you format these a little better so they're more readable? Do like what you did above.